frontend/components/bc/input/BcInputCheckbox.vue

```diff
@@ -0,0 +1,58 @@
+<script lang="ts" setup>
+  import {
+    faInfoCircle
+  } from '@fortawesome/pro-regular-svg-icons'
+  import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+  import type { BcInputError } from '~/components/bc/input/BcInputError.vue';
+
+  const props = defineProps<{
+    error?: BcInputError,
+    label?: string,
+    infoText?: string,
+  }>()
+
+  const id = props.label ? useId() : undefined
+  const input = defineModel<boolean>()
+
+</script>
+
+<template>
+  <BcInputError :error>
+  <span>
+    <label 
+      v-if="label"
+      class="label"
+      :for="id"
+    >
+      {{ label }}
+    </label>
+    <BcTooltip 
+      v-if="infoText"
+      class="bc-input-checkbox__info"
```

Comment:
I was using [BEM](https://getbem.com/) here. Which makes it a bit easier to read, as in the pattern:
`my-component-name-block`_`element`--`modifier`, the `element` and the `modifier` sticks out a little bit.

Block helps a bit with debugging and finding the right component:
![image](https://github.com/user-attachments/assets/defee769-79df-45ba-8cc2-e0ec62ae157f)
(but `dev tools` are also quite good these days)

Reply:
I personally don't like the aesthetics when mixing `-` `_`and `--`  ...but if it help fine by me. 
But in the future we should first discuss, before introducing new naming patters and if we agree to use it all should stick with it otherwise it brings more chaos then helps.